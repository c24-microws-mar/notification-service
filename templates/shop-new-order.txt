New order

A new order has placed

Send the items of the order to:
{%name%}
{%address%}

Items of the order:

{%items%}
