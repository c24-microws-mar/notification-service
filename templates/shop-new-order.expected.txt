New order

A new order has placed

Send the items of the order to:
Lucy Cechtelar
426 Jordy Lodge
Cartwrightshire, SC 88120-6700

Items of the order:

The Offspring - The Offspring 15.00
Smash - The Offspring 20.00
